build-configs/configs/BuildConfigType.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import type { ThemeType } from '../themes/ThemeType'
+import type { FeatureFlagsType } from './featureFlags'
+
+export type BuildConfigType = {|
+  appTitle: string,
+  cmsUrl: string,
+  switchCmsUrl?: string,
+  featureFlags: FeatureFlagsType,
+  theme: ThemeType,
```

Comment:
I'd call it `brightTheme` in contrast to the darkTheme (no pun intended)

Reply:
Maybe rather lightTheme?

Reply:
Even better :)